app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtActionButtons.java

```diff
@@ -208,7 +199,36 @@ public boolean onActionLongClick(final @StringRes int action) {
                 }
                 return true;
             }
-            case R.string.abid_todotxt_current_date: {
+            case R.string.abid_todotxt_priority: {
```

Comment:
Could you describe the behaviour you are trying to achieve here?

Reply:
In my experience, when adding a new task below another, the task's priority would usually be the same.
The goal is to set/unset the priority to be the same as the top one.

This is what felt right to me for multi line:
- if at least one priority is different than the top one - set all to the top priority
- if they are the same and same as the top one - remove priorities as there is nothing else to do (another press restores it)

That also means, that any priorities can be removed with at most two presses.
Might also add a check for the bottom line as well.

Reply:
So long pressing the priority button copies the priority from the first task to all the others? 

Does this really need a long press? Currently one can select all the tasks and assign them a priority pretty easily.

You can select multiple tasks by simply creating a selection covering multiple tasks in the editor or by using the search dialog.

So search -> select all -> priority action -> select priority to assign to all selected tasks.

Reply:
> So long pressing the priority button copies the priority from the first task to all the others?

By "top" I mean the previous line from selection. Priority is copied to the selected lines. 
The exact behavior is not set in stone.

> Does this really need a long press? Currently one can select all the tasks and assign them a priority pretty easily.

In my experience, it feels much easier and responsive in daily use, as it makes creating another task a little bit faster than the other methods.

Reply:
Ah.

So the current workflow is:

select -> priority action -> choose

And the new workflow for this use case is

select -> long press priority action 

We're replacing 2 short presses with one long press.

Reply:
Well said